include/kernels/dslash_mobius_eofa.cuh

```diff
@@ -70,6 +68,12 @@ namespace quda
         inv(inv_),
         sherman_morrison(sherman_morrison_)
       {
+        if (out.size() > max_n_src) errorQuda("vector set size %lu greater than max size %d", out.size(), max_n_src);
```

Comment:
I think you can get rid of this check now that you moved the recursive decomposition into the instantiate routines

Reply:
[9a98546](https://github.com/lattice/quda/pull/1465/commits/9a98546a30b343a906a101d89215ab3c8322775c)